mne/preprocessing/ica.py

```diff
@@ -118,132 +120,110 @@ def _check_for_unsupported_ica_channels(picks, info):
 
 
 class ICA(ContainsMixin):
-    """M/EEG signal decomposition using Independent Component Analysis (ICA).
+    u"""M/EEG signal decomposition using Independent Component Analysis (ICA).
 
-    This object can be used to estimate ICA components and then
-    remove some from Raw or Epochs for data exploration or artifact
-    correction.
+    This object can be used to estimate ICA components and then remove some
+    from Raw or Epochs for data exploration or artifact correction.
 
-    Caveat! If supplying a noise covariance keep track of the projections
+    Caveat! If supplying a noise covariance, keep track of the projections
     available in the cov or in the raw object. For example, if you are
     interested in EOG or ECG artifacts, EOG and ECG projections should be
-    temporally removed before fitting the ICA. You can say::
+    temporally removed before fitting ICA, for example::
 
         >> projs, raw.info['projs'] = raw.info['projs'], []
         >> ica.fit(raw)
         >> raw.info['projs'] = projs
 
-    .. note:: Methods implemented are FastICA (default), Infomax,
-              Extended-Infomax, and Picard. Infomax can be quite sensitive to
-              differences in floating point arithmetic. Extended-Infomax seems
+    .. note:: Methods currently implemented are FastICA (default), Infomax,
+              Extended Infomax, and Picard. Infomax can be quite sensitive to
+              differences in floating point arithmetic. Extended Infomax seems
               to be more stable in this respect enhancing reproducibility and
               stability of results.
-              The stopping criteria of FastICA, Infomax, Extended
-              Infomax and Picard differ, making it hard to compare different
-              tolerance levels, but a rule of thumb is
-              `tol_fastica = tol_picard ** 2`. Reducing the tolerance speeds up
-              estimation but the consistency of the obtained results decreases.
```

Comment:
This comment on Picard must have been recent. Why is it not good?

Reply:
I think it's in Notes now

Reply:
Yes, I think this comment is too specific for the ICA docstring.

Reply:
sorry but I don't see the comment anywhere. Do I miss something?

Reply:
I just looked and I can't find it either! @cbrnr can you move this to the `Notes` section

Reply:
Wait, which notes section are we talking about? I though this comment would best fit somewhere in the implementation of Picard. Here, I don't think it adds much clarification, especially since Infomax doesn't even have a `tol` parameter.

Reply:
Notes of the ICA class (if it doesn't exist, add it)

Reply:
OK, I'll create one, but is anyone up to rephrasing it? I think Picard is more or less an Extended FastICA, so I'm not sure if the tolarance can really be compared with Infomax?

Reply:
Reducing the tolerance speeds up estimation at the cost of consistency of the obtained results. It is difficult to directly compare tolerance levels between Infomax and Picard, but for Picard and FastICA a good rule of thumb is `tol_fastica = tol_picard ** 2`.

Reply:
Thanks, I like it! Moving that to the Notes section.

Reply:
Sounds good to me as well!